test/integration/master/synthetic_master_test.go

```diff
@@ -224,6 +225,101 @@ func TestStatus(t *testing.T) {
 		}
 	}
 }
+func constructBodyByManyLabels(val string, size int) string {
+
+	var temp = `{
+		"apiVersion": "apps/v1beta1",
+		"kind": "Deployment",
+		"metadata": {
+		   "name": "test-deployment2",
+		   "namespace": "default",
+		   "labels": {`
+
+	for i := 0; i < size; i++ {
+		temp += "\"test" + strconv.Itoa(i) + "\" : \"" + val + "\" ,"
```

Comment:
Better use a `bytes.Buffer`. Constructing strings with `+=` is quadratic in runtime. Compare http://herman.asia/efficient-string-concatenation-in-go.

Reply:
Updated.